Src/LoaderHelpers.h

```diff
@@ -221,6 +221,37 @@ namespace DirectX
             }
         }
 
+        //--------------------------------------------------------------------------------------
+        inline DXGI_FORMAT StripSRGB(_In_ DXGI_FORMAT format) noexcept
```

Comment:
I'm debating if calling this ``MakeLinear`` might be better than ``StripSRGB``?

That said, it's not clear that without ``_SRGB`` it is always linear...

Reply:
I'm going to make it ``MakeLinear`` and merge.